cmd/virt-launcher/virt-launcher.go

```diff
@@ -303,6 +325,7 @@ func cleanupContainerDiskDirectory(ephemeralDiskDir string) {
 }
 
 func main() {
+	domainCreationTimeout := pflag.Duration("domain-creation-timeout", defaultStartTimeout, "Amount of time to wait for libvirt domain creation")
```

Comment:
does it make sense to call this `qemu-creation-timeout` or to rename/create an alias to `qemu-timeout` to be `domain-timeout`?

Reply:
Good point, "domain-creation" is too vague, I'll prefix with "libvirt-" so it's obvious which component the option is about.